app/forms/curation_concerns/generic_work_form.rb

```diff
@@ -8,7 +8,7 @@ class GenericWorkForm < Umrdr::Forms::WorkForm
     self.required_fields = [ :title, :creator, :methodology, :description, :rights, :subject ]
 
     def rendered_terms
-      [ :title, :creator, :methodology, :description, :date_coverage, :rights, :subject, :keyword, :language, :resource_type, :isReferencedBy ]
+      [ :title, :creator, :methodology, :description, :date_coverage, :rights, :subject, :keyword, :language, :resource_type, :isReferencedBy, :visibility ]
```

Comment:
That would mean adding another metadata field, and I think we decided that we are using visibility machinery to save as a draft